docs/en/wallet-attestation.rst

```diff
@@ -59,6 +59,12 @@ Wallet Instance Initialization and Registration
 
 **Step 1:**: The User starts the Wallet Instance mobile app for the first time.
 
+.. note::
+
+  The WP MUST verify the WI by using the app store vendors API, for Android is Play Integrity API and for iOS is DeviceCheck, these services are defined in this specification as **Device Integrity Service (DIS)**.
```

Comment:
a requirement must not be defined in a note but within the list of the requirements

this can be only applied to local integrated WSCD, therefore this is something to be specialized in that context and not widely

Reply:
I can append it to the fourth requirement already defined.

Reply:
please do.